dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/bnd.bnd

```diff
@@ -17,35 +17,23 @@ src: \
   test-applications/addNumbersClient/src,\
   test-applications/calculator/src,\
   test-applications/converter/src,\
-  test-applications/ejbServiceRefBndAppCodes/src,\
```

Comment:
What happened with these apps that they're being removed?

Reply:
We moved these apps to com.ibm.ws.jaxws.2.2.webcontainer_fat_extended bundle